Userland/Applications/Browser/History.h

```diff
@@ -18,6 +18,7 @@ class History {
         DeprecatedString title;
     };
     void dump() const;
+    Vector<URLTitlePair> get_all_history_entries();
```

Comment:
why is this not just a getter for `m_items`?

Reply:
Fixed